communication/include/communication/CommDefs.hpp

```diff
@@ -46,6 +46,14 @@ static constexpr size_t MSG_HEADER_SIZE = sizeof(Header);
 typedef std::unordered_map<NodeNum, NodeInfo> NodeMap;
 
 enum CommType { PlainUdp, SimpleAuthUdp, PlainTcp, SimpleAuthTcp, TlsTcp, TlsMultiplex };
+const static std::unordered_map<CommType, const char *> commTypeToName = {
```

Comment:
In general, we prefer not to add to the PR things that are not directly related to the PR.
(every improvement is blessed, but it is better to have only the relevant changes in the PR)